pages/common/mlr.md

```diff
@@ -0,0 +1,23 @@
+# mlr
+
+> Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON.
+
+- Pretty-print a CSV file to a tabular format:
+
+`mlr --icsv --opprint cat {{example.csv}}`
+
+- Sort first alphabetically on one field, and secondarily sort numerically descending on another field:
```

Comment:
Perhaps `1` instead of `one` will read better?

Also, perhaps `sort in descending numerical order` would read better.

Reply:
I'm thinking of changing this line up entirely to something more like:

> Sort first alphabetically on field1, and then sort in descending numerical order on field2

as the code sample reads:

`mlr --icsv --opprint sort -f {{field1}} -nr {{field2}} {{example.csv}}`

Thoughts?

Reply:
I've implemented that change in my latest commit, please let me know if you dislike the update and I'll go with your initial suggestion. Thank you.